### PR TITLE
Feature/share weight

### PR DESCRIPTION
**Pull request by DriesMeerman**
* Restructured code to separate styling and logic into separate files.
* Refactored handleClick so Dom manipulation and logic are more split up

* Added weights to the random person selection so if you are picked you are less likely to be picked again.
* Included link to website on readme

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,224 +8,8 @@
...
-        .toggle-switch-switch {
-        right: 0%;
-      }
-
-      .toggle-text {
-        margin: 0 10px;
-      }
-    </style>
+    <script src="/script.js"></script>
+    <link rel="stylesheet" href="/style.css" />
```
Comment (MichielvanBeers) [inline index.html]: @DriesMeerman could you remove the '/' before script.js and style.css? In Chrome it fails to load them for some reason

**Comment by MichielvanBeers** [comment]
Haha great stuff Dries! One small remark and then we are good to go. 💃🏻

**Comment by DriesMeerman** [comment]
> Haha great stuff Dries! One small remark and then we are good to go. 💃🏻

Sure what exactly is the remark?

**Comment by MichielvanBeers** [comment]
I put a comment on the import of the script and style sheet, since it didn't load correctly for me when using chrome. If you go to the could, you should be able to see it in line

**Comment by DriesMeerman** [comment]
> I put a comment on the import of the script and style sheet, since it didn't load correctly for me when using chrome. If you go to the could, you should be able to see it in line

image
I can't see your comment. 
I attempted to get the same error but could not reproduce it if you have any details please let me know.

**Review by MichielvanBeers — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (DriesMeerman) [reply index.html]: updated

**Review by MichielvanBeers — APPROVED** [review]
🥳

**Pull request merged**